src/modules/collection/utils.ts

```diff
@@ -81,7 +81,8 @@ export function getCollectionSymbol(collection: Collection) {
 
 export function toCollectionObject(collections: Collection[]) {
   return collections.reduce((obj, collection) => {
-    obj[collection.id] = utils.omit<Collection>(collection, ['items'])
+    const { items: _, ...rest } = collection as Collection & { items?: any[] }
```

Comment:
Does this work?
```suggestion
    const { items: _, ...rest } = collection as Omit<Collection, 'items'>
```

Reply:
It's not an omit actually, we are adding an optional property to the type (`Collection` doesn't have  an `items` property)